blueocean-material-icons/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@jenkins-cd/blueocean-material-icons",
+  "version": "0.0.4",
+  "description": "Material icons for the Jenkins Design Language",
+  "main": "dist/js/components/index.js",
+  "scripts": {
+    "lint": "gulp lint",
+    "gulp": "gulp build",
+    "mvnclean": "gulp clean",
```

Comment:
this isn't actually being built by maven, i'd clean this `package.json` up to have only the minimum stuff